### PR TITLE
chore: docs homepage

### PR DESCRIPTION
**Pull request by fsansalvadore**
## What kind of change does this PR introduce?

Screenshot 2023-05-10 at 18 30 22
Screenshot 2023-05-10 at 18 30 30

### CODE REVIEW

**Comment by gregnr** [comment]
@fsansalvadore Looks awesome 🤩

I'm getting a double scroll sometimes (mainly when scrolled to the top):

https://github.com/supabase/supabase/assets/4133076/858378a6-29c6-4bff-baad-a0b13747182f

File: apps/docs/layouts/HomeLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            modifiedTime: new Date().toISOString(),
+            authors: ['Supabase'],
+          },
+        }}
+      />
+      <HomePageCover meta={props.meta} />
+      <LayoutMainContent>
+        <div className={['relative transition-all ease-out', 'duration-150 '].join(' ')}>
+          <article className="prose dark:prose-dar max-w-none">
+            {/* <h1>{props.meta.title} 🦸</h1> */}
```
Comment (gregnr) [inline apps/docs/layouts/HomeLayout.tsx:68]: 👀 no longer needed?

Comment (fsansalvadore) [reply apps/docs/layouts/HomeLayout.tsx]: nope, thanks!

File: apps/docs/components/Navigation/NavigationMenu/HomeMenuIcons.tsx [hunk 2]
```diff
@@ -23,18 +23,32 @@ export function IconMenuHome({ width = 16, height = 16 }: HomeMenuIcon) {
...
+    //   fill="none"
+    //   xmlns="http://www.w3.org/2000/svg"
+    // >
+    //   <path
+    //     d="M2.923 10.927A5.833 5.833 0 0 1 2.14 8m2.877-5.046A5.833 5.833 0 0 1 8 2.14m5.13 3.025c.465.84.73 1.807.73 2.835m-2.883 5.049A5.832 5.832 0 0 1 8 13.859m6.172-10.027a2 2 0 1 1-4 0 2 2 0 0 1 4 0ZM5.81 12.148a2 2 0 1 1-4 0 2 2 0 0 1 4 0Zm8.362 0a2 2 0 1 1-4 0 2 2 0 0 1 4 0ZM5.81 3.832a2 2 0 1 1-4 0 2 2 0 0 1 4 0Z"
+    //     stroke="currentColor"
+    //     strokeMiterlimit="10"
+    //     strokeLinejoin="bevel"
+    //   />
+    // </svg>
```
Comment (gregnr) [inline apps/docs/components/Navigation/NavigationMenu/HomeMenuIcons.tsx:39]:
```suggestion
```

File: apps/docs/components/Navigation/NavigationMenu/HomeMenuIcons.tsx [hunk 3]
```diff
@@ -23,18 +23,32 @@ export function IconMenuHome({ width = 16, height = 16 }: HomeMenuIcon) {
...
+        stroke="#A0A0A0"
+        stroke-miterlimit="10"
+        stroke-linejoin="bevel"
```
Comment (gregnr) [inline apps/docs/components/Navigation/NavigationMenu/HomeMenuIcons.tsx:51]: Confirming these work in JSX? Or:
```suggestion
        strokeMiterlimit="10"
        strokeLinejoin="bevel"
```

File: apps/docs/components/Navigation/NavigationMenu/HomeMenuIcons.tsx [hunk 4]
```diff
@@ -404,17 +418,17 @@ export function IconMenuKotlin({ width = 16, height = 16 }: HomeMenuIcon) {
...
     >
       <path
-        stroke="currentColor"
-        strokeLinejoin="bevel"
-        strokeMiterlimit="10"
-        d="M8.99 1.133a7.866 7.866 0 1 0 6.133 2.94l-6.77 6.77-1.956-1.958m2.67-4.688a4.789 4.789 0 1 0 3.98 2.124"
+        d="M8.9895 1.13281C4.64498 1.13281 1.12305 4.65474 1.12305 8.99927C1.12305 13.3438 4.64498 16.8657 8.9895 16.8657C13.334 16.8657 16.856 13.3438 16.856 8.99927C16.856 7.13428 16.207 5.42087 15.1225 4.07258L8.35267 10.8424L6.39571 8.88546M9.06632 4.19659C6.42145 4.19659 4.27737 6.34068 4.27737 8.98555C4.27737 11.6304 6.42145 13.7745 9.06632 13.7745C11.7112 13.7745 13.8553 11.6304 13.8553 8.98555C13.8553 7.99916 13.5571 7.08243 13.0459 6.32059"
+        stroke="#A0A0A0"
+        stroke-miterlimit="10"
+        stroke-linejoin="bevel"
```
Comment (gregnr) [inline apps/docs/components/Navigation/NavigationMenu/HomeMenuIcons.tsx:431]: Ditto
```suggestion
        strokeMiterlimit="10"
        strokeLinejoin="bevel"
```

File: apps/docs/layouts/DefaultLayout.tsx [hunk 5]
```diff
@@ -60,26 +60,32 @@ const Layout: FC<Props> = (props: Props) => {
...
-        <div
-          className={[
-            'border-scale-400 dark:bg-scale-200 table-of-contents-height border-l',
-            'thin-scrollbar overflow-y-auto sticky hidden md:block md:col-span-3 px-2',
-          ].join(' ')}
-        >
-          <TableOfContents toc={props.toc} video={props.meta?.video} />
+      <LayoutMainContent>
+        <div className={['relative transition-all ease-out', 'duration-150 '].join(' ')}>
+          {/* <p className="text-brand-900 tracking-wider">Tutorials</p> */}
```
Comment (gregnr) [inline apps/docs/layouts/DefaultLayout.tsx:65]:
```suggestion
```

File: apps/docs/layouts/DefaultLayout.tsx [hunk 6]
```diff
@@ -60,26 +60,32 @@ const Layout: FC<Props> = (props: Props) => {
...
+          <article className="prose dark:prose-dar max-w-none">
+            <h1>{props.meta.title}</h1>
+            {/* <div className="max-w-xs w-32 h-[1px] bg-gradient-to-r from-brand-800 to-brand-900 my-16"></div> */}
```
Comment (gregnr) [inline apps/docs/layouts/DefaultLayout.tsx:68]:
```suggestion
```

File: apps/docs/layouts/guides/index.tsx [hunk 7]
```diff
@@ -105,74 +106,71 @@ const Layout: FC<Props> = (props) => {
...
+              <div className="border-l">
+                <span className="block font-mono text-xs uppercase text-scale-1200 px-5 mb-6">
+                  On this page
+                </span>
+                <GuidesTableOfContents list={tocList} />
+              </div>
+            </div>
+          )}
+        </div>
+        {/* <FooterHelpCallout footerHelpType={props.meta?.footerHelpType} title={props.meta?.title} /> */}
```
Comment (gregnr) [inline apps/docs/layouts/guides/index.tsx:172]:
```suggestion
```

File: packages/ui/index.tsx [hunk 8]
```diff
@@ -15,6 +15,7 @@ export * from './src/components/Icon'
 export * from './src/components/Icon/IconContext'
 export * from './src/components/GlassPanel'
 export * from './src/components/IconPanel'
+export * from './src/components/Icon/IconBackground'
 // export * from './src/components/Image'
```
Comment (gregnr) [inline packages/ui/index.tsx:19]:
```suggestion
```

File: apps/docs/pages/index.mdx [hunk 9]
```diff
@@ -227,138 +202,150 @@ export const products = [
...
+  {
+    title: 'C#',
+    icon: 'reference-csharp',
+    href: 'reference/csharp/introduction',
+  },
+  {
+    title: 'Swift',
+    icon: 'reference-swift',
+    href: 'reference/swift/introduction',
+  },
```
Comment (gregnr) [inline apps/docs/pages/index.mdx:319]: Just added Kotlin yesterday - can we add here?

**Review by gregnr — APPROVED** [review]

File: apps/docs/components/Navigation/NavigationMenu/HomeMenuIcons.tsx (hunk 3 above)
Comment (saltcod) [reply apps/docs/components/Navigation/NavigationMenu/HomeMenuIcons.tsx:51]: wish there was a prettier option for these...

**Pull request merged**